Sources/FluentKit/Model/EagerLoad.swift

```diff
@@ -33,3 +39,21 @@ public protocol EagerLoadable {
         Loader.Model == To,
         Builder.Model == From
 }
+
+extension EagerLoadable {
+    public static func eagerLoad<Builder>(
+        _ relationKey: KeyPath<From, Self>,
+        to builder: Builder
+    ) where Builder: EagerLoadBuilder, Builder.Model == From {
+        return Self.eagerLoad(relationKey, withDeleted: false, to: builder)
+    }
+    
+    // Default non functional implementation for non breaking API change
```

Comment:
I knew about the potential for infinite recursion but I though the purpose was just to prevent major change to the API. I'll add the function to all conforming types instead.